src/VernaContextProvider.tsx

```diff
@@ -66,31 +86,121 @@ function genDefaultUiSchema(schema: JSONSchema7, uiSchema: UiSchema) {
   return defaultUi;
 }
 
+function getSelectedSchema(
+  defaultSchema: JSONSchema7,
+  defaultSelector: string | undefined,
```

Comment:
From my point of view, in the scope of this function you could name these arguments `schema` and `selector`. Currently I found it confusing we variable names existing in the upper scope.

Reply:
It's actually the same constant that is used here, I use defaultSchema & defaultSelector in it because it's called only on mounting the context provider, those functions are just for initializing